src/parser/__tests__/parser-basic-test.js

```diff
@@ -19,7 +19,7 @@ describe('basic', () => {
         value: 'a',
         kind: 'simple'
       },
-      flags: []
+      flags: '',
```

Comment:
I think empty string is better.

Most common usages: check whether flag is enabled, remove flag, add flag. All three would require null checks (meh) and docs pointing that out. Furthermore, we are mimicking `RegExp#flags`, it returns an empty string.